nightfall-client/src/services/commitment-storage.mjs

```diff
@@ -688,3 +688,38 @@ export async function findUsableCommitmentsMutex(
     findUsableCommitments(compressedZkpPublicKey, ercAddress, tokenId, _value, onlyOne),
   );
 }
+
+/**
+ * @function getCommitmentsByCompressedZkpPublicKeyList do the role of a service taking care of the
+ * business logic and of a repository doing the communication with the database for this
+ * use case.
```

Comment:
Is this still accurate? Im not sure if the docs refer to the other approach we discussed previously :thinking:

Reply:
Docs are fine.